lib/bot.js

```diff
@@ -111,6 +111,7 @@ class Bot {
   parseText(message) {
     const text = message.mentions.reduce((content, mention) => (
       content.replace(`<@${mention.id}>`, `@${mention.username}`)
+             .replace(`<@!${mention.id}>`, `@${mention.username}`)
```

Comment:
Cool! Maybe the line before isn't needed anymore then (maybe Discord changed something)?